test/unit/spec/MfaVerify_spec.js

```diff
@@ -162,6 +164,8 @@ function (Q,
     var setupOktaPush = _.partial(setup, resAllFactors, { factorType: 'push', provider: 'OKTA' });
     var setupOktaTOTP = _.partial(setup, resVerifyTOTPOnly, { factorType: 'token:software:totp' });
     var setupWebauthn = _.partial(setup, resAllFactors, {  factorType: 'webauthn', provider: 'FIDO' });
+    var setupWebauthnOnly = _.partial(setup, [resRequiredWebauthn, resChallengeWebauthn, resSuccess],
```

Comment:
Debugging the code I found, that in time when `Expect.waitForMfaVerify()` is done, the controller is already rendered and the `model.save()` is called.
So, to prevent the ajax error, when we make the request without the mock, I should pass all three responses.